src/configs/env_var.rs

```diff
@@ -23,6 +24,7 @@ impl<'a> Default for EnvVarConfig<'a> {
             variable: None,
             default: None,
             format: "with [$env_value]($style) ",
+            description: "<env_var module>",
```

Comment:
Is this good? There is still also the previously used static description. Technically we could try generating a better one if the description is unset since unlike a custom module, we do know the variable name unlike an arbitrary shell command, for example to print something more descriptive.